ReleaseProcessScript/tools/Core/git_base_functions.ps1

```diff
@@ -22,14 +22,48 @@ function Get-Current-Branchname ()
   return $(git symbolic-ref --short -q HEAD)
 }
 
-function Get-Last-Version-Of-Branch-From-Tag-Exists ($Branchname)
+function Get-Last-Version-Of-Branch-From-Tag-Exists
+(
+  $From, #A commit-ish from wich to start looking for version tags (backwards in history). Can be a tag name, a branch name, a commit hash, HEAD etc. Default: HEAD
```

Comment:
I cannot reply to the original discussion so... 

In this particular context, the "FromCommitish" might be overkill and makes if harder to read.  I don't think we have a maintenance issue, leaving the names / strings as they are.

I do not like the fact that there's no whitespace between the comment-hash and the comment text. This just reads strange.

Reply:
Do I understand you correctly that the comments (as they are) are acceptable? Including the "commit-ish" in the comment text?

I do not like the missing whitespace between comment-hash and comment text either. However, that's how the comments look like across all files, I just stuck to the style. Do you want me to do a global search & replace (`"#X" -> "# X"`), replace it only for the comments in this file or leave it as it is?

Reply:
okay, fine. we already have comments that are stuck to the hash. okay. Perhaps a cleanup task would be appreciated by the maintainers, then...

And yes. text itself is done. Please re-request once all comments are done. (don't want to check this myself)